spring-cloud-deployer-cloudfoundry/pom.xml

```diff
@@ -13,20 +13,10 @@
 		<relativePath>..</relativePath>
 	</parent>
 
-	<properties>
-		<java.version>17</java.version>
-		<!-- @TODO boot3 updates need for jakarta ee -->
-		<cloudfoundry-java-lib.version>5.12.1.RELEASE</cloudfoundry-java-lib.version>
-		<pivotal-cf-client-reactor.version>2.2.0.RELEASE</pivotal-cf-client-reactor.version>
-		<commons-compress.version>1.26.2</commons-compress.version>
-        <okhttp.version>3.14.9</okhttp.version>
-	</properties>
-
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-deployer-spi</artifactId>
-			<version>${project.version}</version>
```

Comment:
Not needed as it is specified in spring-cloud-deployer-dependencies which is in dep mgmt of root pom.xml